main/models/Step1/character-sheet.ts

```diff
@@ -0,0 +1,31 @@
+import { CharacterClass } from './character-class';
+import { StatsSheet } from './stats-sheet';
+import { Item } from './item';
+
+export class CharacterSheet {
+  public name: string;
+  public sheet_level: number;
+  public stats: StatsSheet;
+  public character: CharacterClass;
+  public inventory: Item[];
+
+  constructor(name: string, level: number, stats: StatsSheet, character: CharacterClass, inventory: Item[]) {
+    this.name = name;
+    this.sheet_level = level;
+    this.stats = stats;
+    this.character = character;
+    this.inventory = inventory;
+  }
+
+  calcHealth(): number {
+    return this.character.class_dice.roll() * this.character.class_level + this.stats.CON;
+  }
+
+  calcArmorClass(): number {
+    return 10 + this.stats.DEX;
```

Comment:
make the assumption that you can have only one armoritem type in your inventory and
- if you have none => default to what you have wrote (10 + this.stats.DEX)
- if you have one => return item armor class